packages/ra-core/src/reducer/admin/resource/list/totalAll.js

```diff
@@ -0,0 +1,26 @@
+import {
+    CRUD_GET_ONE_SUCCESS,
+    CRUD_GET_LIST_SUCCESS,
+    CRUD_DELETE_OPTIMISTIC,
+    CRUD_DELETE_MANY_OPTIMISTIC,
+} from '../../../../actions/dataActions';
+
+export default resource => (previousState = 0, { type, payload, meta }) => {
+    if (!meta || meta.resource !== resource) {
+        return previousState;
+    }
+    if (type === CRUD_GET_ONE_SUCCESS) {
+        return previousState == 0 ? 1 : previousState;
+    }
+    if (type === CRUD_GET_LIST_SUCCESS) {
+        console.log('totalAll:', payload)
```

Comment:
Not sure your policy on committed console logs at this point, but I figured I’d point it out